api/GPUCommandEncoder.json

```
@@ -472,6 +472,49 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "offset_size_optional": {
+          "__compat": {
+            "description": "Offets and `size` optional when copying entire buffer.",
```

For consistency with [other optional parameters](https://github.com/search?q=repo%3Amdn%2Fbrowser-compat-data+%2F%22%28.*%3F%29_optional%22%3A+%5C%7B%2F&type=code):

```suggestion
        "offset_and_size_parameter_optional": {
          "__compat": {
            "description": "Offset and `size` parameters are optional",
```